pkg/nsxt/config/config_ini_legacy_test.go

```diff
@@ -53,3 +53,24 @@ func TestINIValidateUserConfig(t *testing.T) {
 	err = cfg.ValidateConfig()
 	assert.Nil(t, err)
 }
+
+func TestINIValidateCertConfig(t *testing.T) {
```

Comment:
it might be more clearer to use a test case table:
e.g. 
```
func TestINIValidateCertConfig(t *testing.T) {
	testCases := []struct {
		name               string
		cfg                *NsxtConfigINI
                expectedErrMessage string
	}{
	    {
			name:       "empty client cert file",
			cfg:    &NsxtConfigINI{
		                   ClientAuthKeyFile: "client-key",
	                        },
			expectedErrMessage: "client cert file is required if client key file is provided",
		},
		{
			name:       "empty client key file",
			cfg:    &NsxtConfigINI{
		                     ClientAuthCertFile: "client-cert",
	                            },
			expectedErrMessage: ...,
		},
		... // can add more test cases
	}

	for _, testCase := range testCases {
		err := testCase.cfg.ValidateConfig()
		assert.EqualError(t, err, testCase.errorMessage)
	}
}
```